test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -49,6 +51,18 @@ def hook(context):
 
     if context.userData['clientConfigFile'] == '':
         context.userData['clientConfigFile']='-'
+        
+    req = urllib.request.Request(
+        os.path.join(context.userData['middlewareUrl'], 'init',''),
```

Comment:
why is there empty string in the end?

Reply:
So that it would end with `/`

Reply:
why do we need to have the URL end with `/`
also you can just replace `init` with `init/` if you want that